### DISCUSSION TITLE
create a row in a "credit" row on signup

### DISCUSSION THREAD

**Discussion by JSmithOner** [supabase/supabase#29829] [Questions] [answered]
I would like to create a new row in the "credit" table on each new signup but the auth table is protected.How can I do that?
Thanks in advance

**Comment by GaryAustin1** [comment, marked as answer]
You can add a trigger to call a function with the SQL editor.
This is shown in the user management startup guides.
https://supabase.com/docs/guides/auth/managing-user-data#accessing-user-data-via-api

**Reply by JSmithOner** [reply]
thanks found out in the meantime
